package.json

```diff
@@ -14,17 +14,17 @@
     "lint": "eslint . --ext .ts",
     "lint:fix": "eslint .  --ext .ts --fix",
     "format": "prettier --write . && npx prisma format",
-    "start:dev": "cross-env NODE_ENV=development NEW_RELIC_ENABLED=false ts-node src/server.ts",
-    "start:prod": "tsc && cross-env NODE_ENV=production node dist/src/server.js",
+    "start:dev": "cross-env NODE_ENV=development ts-node -r dotenv/config -r newrelic src/server.ts",
```

Comment:
what if I want to disable newrelic locally?

Reply:
It ought to respect `NEW_RELIC_ENABLED` env var https://docs.newrelic.com/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#agent-enabled